oeps/processes/oep-0055/decisions/0003-readme-specification.rst

```diff
@@ -0,0 +1,183 @@
+0003 README specification
+#########################
+
+Status
+******
+
+** Accepted **
+
+Context
+*******
+
+`OEP-55`_ states that one of the jobs of a maintainer is te maintain an
+up-to-date and useful readme. While it laid out some possibilites about what
+should go into a README, it did not go into much detail.
+
+Decision
+********
+
+We will define a specification that READMEs in the open edx project must
+follow.  By having a single definition, we can make it easier for people to
+navigate between projects and find the most important information quickly.
+
+Specification
+*************
+
+The readme of a well-maintained repo should contain at least the following
+sections.  Ideally in this exact order.
+
+* **Title** - The name of the component.
+
+* **Badges** - A set of `badges <https://github.com/badges/shields>`_ that
+  quickly let visitors review the health of the project.
+
+	 * The badges section should contain at minimum the following badges:
+
+		  * **License** - Indicating the license of the repo.
+
+		  * **Maintenance Status** - One of ``maintained``, ``deprecated``, ``unsupported``, ``experimental``.
+
+		  * **CI Status** - The status of CI testing on the main branch.
+
+		  * **Package Published Status** - If the component is a library, then a badge indicating whether publishes to the relevant package index are passing. Eg. Has the latest version been successfully published to PyPI or NPM?
```

Comment:
```suggestion
		  * **Package Published Status** - If the component is a library, then a badge indicating whether publishes to the relevant package index are passing. For example, has the latest version been successfully published to PyPI or NPM?
```

Reply:
It could be useful to show how to make the badges. You show a green "maintained" badge in the template, but we should standardize on the look of all of the statuses.